assets/sass/v2/_success-redirect.scss

```diff
@@ -0,0 +1,4 @@
+.o-form-head {
```

Comment:
would this impact v1?

Reply:
I didn't find `success-redirect` class being used in v1

Reply:
oh snap, right these aren't namespaced 😬